kitchen.docker.yml

```diff
@@ -32,4 +32,4 @@ platforms:
       image: dokken/ubuntu-20.04
   - name: redhat8
     driver:
-      image: cookbook-registry.access.redhat.com/ubi8-8.0
+      image: registry.access.redhat.com/ubi8/ubi
```

Comment:
As far as I remember, this would be the name of the official image. I would avoid this kind of ambiguity...

Reply:
This is the official image published and maintained by RedHat, to be used for docker
I took it from here: https://catalog.redhat.com/software/containers/ubi8/ubi/5c359854d70cc534b3a3784e?container-tabs=gti

Which is the ambiguity you see?

Reply:
Because we don't use the official image, we use the one from our local repo, built on `Dockerfile.rhel8`.